test/application_system_test_case.rb

```diff
@@ -20,6 +20,11 @@ def perform_enqueued_jobs
     worker = Resque::Worker.new("*")
     worker.work(0.0)
   end
+
+  def perform_enqueued_jobs_async
+    worker = SolidQueue::Worker.new(queues: "*", threads: 2, polling_interval: 0)
+    worker.start
+  end
```

Comment:
I think we don't need this one, right? At least not yet.

Reply:
Yes, but as I said I'm not sure if this adds some value.

I would like to have a test like the `PaginatJobsTest` but with the workers it's more complex.